docs/zh_CN/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md

```diff
@@ -123,7 +123,7 @@
 
 #### 移除过时的功能：
 
-- `Reflect.enumrate` 因为他已经从标准中移除了
+- `Reflect.enumerate` 因为他已经从标准中移除了
```

Comment:
I generally avoid touching non English content, but my tool doesn't care and is actually quite happy to check content that only uses latin letters for code.

As always, happy to drop.